docker/experiment-covid-native.dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG EXPERIMENTS_VERSION
```

Comment:
Do we need two separate Dockerfiles and images for each experiment or can we have one? It's starting to seem like overkill to me. Could we merge the two experiment base containers? The responsibilities for the two are different, but I'm not sure it's worth the extra admin and more difficult development to maintain two images.

We could just have a single `Dockerfile` at the top level of each experiment repo and build a single image.

Afaict the responsibilities of this single container would be the following, which I don't think cause any conflicts:

- Build and run the native version (i.e. check out code, build with standard toolchain, execute)
- Build and upload the wasm version (i.e. check out code, build with Faasm toolchain, upload)

Reply:
Will do this in faasm/experiment-base#7. In the meantime we can take [this Dockerfile](https://github.com/faasm/experiment-lammps/blob/master/Dockerfile) as reference.